sofa-boot-project/sofa-boot-core/rpc-sofa-boot/src/main/java/com/alipay/sofa/rpc/boot/runtime/converter/RpcBindingConverter.java

```diff
@@ -407,8 +407,9 @@ protected void convertServiceAnnotation(RpcBindingParam bindingParam,
                                             SofaService sofaServiceAnnotation,
                                             SofaServiceBinding sofaServiceBindingAnnotation,
                                             BindingConverterContext bindingConverterContext) {
-        bindingParam.setTimeout(sofaServiceBindingAnnotation.timeout());
-
+        if (sofaServiceBindingAnnotation.timeout() != 0) {
```

Comment:
add unit test  to make sure if timeout is 0s, invoke timeout will be. default 3 seconds

Reply:
default timeout is 0,the test is as follows